### PR TITLE
Add LXD logo and minor fix

### PR DESCRIPTION
**Pull request by geaaru**

### CODE REVIEW

File: static/css/common.css [hunk 1]
```diff
@@ -282,8 +282,8 @@ li a[href='https://plus.google.com/+sabayon']::before {
     content: "\f1a0";
 }
 
-a[href='https://atlas.hashicorp.com/Sabayon/boxes/spinbase-amd64']::before {
-    background-image: url("/img/vagrantcloud-logo.png");
+a[href='http://dl.sabayon.org/sabayon/sbi/vagrant/Sabayon_Linux_DAILY_amd64_SpinBase.box']::before {
+    background-image: url('../img/vagrantcloud-logo.png");
```
Comment (Enlik) [inline static/css/common.css:286]: broken

Comment (geaaru) [reply static/css/common.css:286]: ??? 
It is used under cloud/ path so ../img/ it's right imho. I try to create a task to see produced pages under mottainai namespace.

Comment (Enlik) [reply static/css/common.css:286]: Sorry for not being clear enough, I saw it being displayed in red nicely and so thought it was clear enough. Quotes are "broken."

Comment (Ryuno-Ki) [reply static/css/common.css:286]: I can see a single one on the one side, but a double on the other one :o

**Pull request merged**